src/swap/handle_swap_sign_transaction.c

```diff
@@ -119,7 +121,75 @@ bool swap_copy_transaction_parameters(create_transaction_parameters_t* params) {
     return true;
 }
 
-bool swap_check_validity() {
+static address_t* swap_get_tx_recipient_address(void) {
```

Comment:
Thus we can here do `static address_t* swap_get_tx_recipient_address(bool is_jetton_swap) {`

Reply:
Done